Tasks/CopyFilesV2/copyfiles.ts

```diff
@@ -38,22 +38,33 @@ function makeDirP(targetFolder: string, ignoreErrors: boolean): void {
 }
 
 /**
- * Gets stats for the provided path. Will ignore ENOENT error if ignoreEnoent is true.
- * If ignoreEnoent is false ENOENT will be thrown from the function.
- * @param path path for which methid will try to get tl.FsStats.
- * @param ignoreEnoent ignore ENOENT error during check of path stats.
- * @returns 
+ * Gets stats for the provided path.
+ * Will throw error if entry does not exist and `throwEnoent` is `true`.
+ * @param path path for which method will try to get `fs.Stats`.
+ * @param throwEnoent throw error if entry does not exist.
+ * @returns `fs.Stats` or `null`
  */
-function stats(path: string, ignoreEnoent: boolean): tl.FsStats {
-    try {
-        return tl.stats(path);
-    } catch (err) {
-        if (err.code != 'ENOENT' && ignoreEnoent) {
-            throw err;
+function stats(path: string, throwEnoent: boolean = true): fs.Stats | null {
+    if (fs.existsSync(path)) {
+        return fs.statSync(path);
+    } else {
+        const message: string = `Entry "${path}" does not exist`;
+        if (throwEnoent) {
+            tl.warning(message);
```

Comment:
Are we need this warning message here?

Reply:
Yes, it is better to warn the user before throwing an error.